firebase_connector/models/firebase_notification.py

```diff
@@ -66,6 +66,11 @@ class FirebaseNotification(models.Model):
         string="Partner read status of the notification",
         readonly=True,
     )
+    language = fields.Selection([('fr_CH', 'French'),
+                                 ('de_DE', 'German'),
+                                 ('it_IT', 'Italian'),
+                                 ('en_US', 'English')], required=True)
+
```

Comment:
You should use available languages instead of hardcoding them. See https://github.com/CompassionCH/compassion-modules/blob/cb9bb3102ab811a1b0f4d5ff727ccac637526ed1/crm_request/models/request.py#L37 for an example